test-project/Assets/.Schema/improbable/gdk/tests/exhaustive_test.schema

```diff
@@ -117,23 +157,7 @@ component ExhaustiveMapKey {
   map<SomeEnum, string> field18 = 18;
 }
 
-// Blittable subset
-component ExhaustiveBlittableSingular {
-  id = 197720;
-  bool field1 = 1;
-  float field2 = 2;
-  int32 field4 = 4;
-  int64 field5 = 5;
-  double field6 = 6;
-  uint32 field8 = 8;
-  uint64 field9 = 9;
-  sint32 field10 = 10;
-  sint64 field11 = 11;
-  fixed32 field12 = 12;
-  fixed64 field13 = 13;
-  sfixed32 field14 = 14;
-  sfixed64 field15 = 15;
-  EntityId field16 = 16;
-  SomeType field17 = 17;
-  SomeEnum field18 = 18;
-}
+component ExhaustiveMapKey {
+  id = 197719;
+  data ExhaustiveMapKeyData;
+}
```

Comment:
silly question: what's the general guideline on when to have newlines/when not to?

Reply:
Text files should have a newline. I.e. - this should have a newline at the end